Importer/Interpreter/Field/DatetimeFieldInterpreter.php

```diff
@@ -28,6 +28,13 @@ public function interpret(FieldConfigurationInterface $configuration, $value)
         $dateTime = \DateTime::createFromFormat($configuration->getFormat(), $value);
 
         if ($dateTime === false) {
+            $dateTime = \DateTime::createFromFormat('Y-m-d', $value);
```

Comment:
What about the unit test? This is quite important functionality so it makes sense to test it.

And after adding test you could refactor this a bit - `if` inside `if` is not the easiest thing to read. This could be implemented as:

```
$dateTime = // try to import using default format
if (false !== $dateTime) {
    return $dateTime;
}

$dateTime = // try to import using date format
if (false !== $dateTime) {
    // set time to 0
    return $dateTime;
}

// throw exception
```